lib/cartopy/mpl/ticker.py

```diff
@@ -175,12 +297,18 @@ def __init__(self,
                  dateline_direction_label=False,
                  degree_symbol=u'\u00B0',
                  number_format='g',
-                 transform_precision=1e-8):
+                 transform_precision=1e-8,
+                 decimal=False,
```

Comment:
Same case as above; this is a breaking change according to the tests, so you must address it by either flipping the default `decimal` value or targetting a v1.0.0 branch.

Reply:
Set to True by default (until v1.0).